### PR TITLE
feat(shinigami-eyes): init

### PR DESCRIPTION
**Pull request by jh-devv**
## 🎉 Theme for [Shinigami Eyes](https://github.com/shinigami-eyes/shinigami-eyes) 🎉

preview

A Chrome/Firefox addon that highlights transphobic/anti-LGBT and trans-friendly subreddits/users/facebook pages/groups with different colors.

For more information: [Shinigami Eyes website](https://shinigami-eyes.github.io/)

* [Shinigami Eyes for Chrome](https://chrome.google.com/webstore/detail/ijcpiojgefnkmcadacmacogglhjdjphj/)
* [Shinigami Eyes for Firefox](https://addons.mozilla.org/en-US/firefox/addon/shinigami-eyes/)

## 💬 Additional Comments 💬

Since it's an extension and just changes some link colors, the changes are bit hard to preview. They also just minor color changes.
I used Google as an example to view some sites it tags.

The [scripts/userstyles.yml file
](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml) also includes an optional icon variable for the userstyle.
Since this extension does not have *proper official icon*, I had to trace it and submit it to Simple icons: https://github.com/simple-icons/simple-icons/pull/11340. 

We can update the metadata with the icon when it's ready.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `preview.webp` - composite image of all four individual flavor screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

**Comment by jh-devv** [comment]
Waiting for merge: https://github.com/simple-icons/simple-icons/pull/11340

File: styles/shinigami-eyes/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  domain(google.de),
+  domain(google.dk),
+  domain(google.es),
+  domain(google.fi),
+  domain(google.fr),
+  domain(google.is),
+  domain(google.it),
+  domain(google.no),
+  domain(google.pt),
+  domain(google.se)
```
Comment (isabelroses) [inline styles/shinigami-eyes/catppuccin.user.css:92]: can we put the domains in quotes
e.g. `domain("google.it")`

**Review by isabelroses — COMMENTED** [review]
LGTM, minor nitpick otherwise good to go

File: styles/shinigami-eyes/catppuccin.user.css (hunk 1 above)
Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css:92]: > can we put the domains in quotes
e.g. domain("google.it")

seems good, i'll do that

File: scripts/userstyles.yml [hunk 2]
```diff
@@ -727,6 +730,14 @@ userstyles:
     readme:
       app-link: "https://github.com/searxng/searxng"
     current-maintainers: [*sekki21956, *ryanccn]
+  shinigami-eyes:
+    name: Shinigami Eyes
+    categories: [browser_extension social_networking]
```
Comment (isabelroses) [inline scripts/userstyles.yml:735]:
```suggestion
    categories: [browser_extension, social_networking]
```

File: scripts/userstyles.yml [hunk 3]
```diff
@@ -727,6 +730,14 @@ userstyles:
     readme:
       app-link: "https://github.com/searxng/searxng"
     current-maintainers: [*sekki21956, *ryanccn]
+  shinigami-eyes:
+    name: Shinigami Eyes
+    categories: [browser_extension, social_networking]
+    #icon: shinigami-eyes
```
Comment (uncenter) [inline scripts/userstyles.yml:736]:
```suggestion
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: scripts/userstyles.yml [hunk 4]
```diff
@@ -727,6 +730,14 @@ userstyles:
...
+    color: pink
```
Comment (uncenter) [inline scripts/userstyles.yml:736]:
```suggestion
    color: mauve
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/shinigami-eyes/catppuccin.user.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document
```
Comment (uncenter) [inline styles/shinigami-eyes/catppuccin.user.css:18]: Please add a comment detailing where this list is from.

File: styles/shinigami-eyes/catppuccin.user.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  domain("google.es"),
+  domain("google.fi"),
+  domain("google.fr"),
+  domain("google.is"),
+  domain("google.it"),
+  domain("google.no"),
+  domain("google.pt"),
+  domain("google.se")
+{
+
```
Comment (uncenter) [inline styles/shinigami-eyes/catppuccin.user.css:94]:
```suggestion
```

File: styles/shinigami-eyes/catppuccin.user.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+
+    @accent-color: @catppuccin[@@lookup][@@accent];
```
Comment (uncenter) [inline styles/shinigami-eyes/catppuccin.user.css:116]: I don't think these should be changed from the template. We may decide to change the colors at some point, etc.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/shinigami-eyes/catppuccin.user.css (hunk 5 above)
Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css:18]: Yup :3

File: styles/shinigami-eyes/catppuccin.user.css (hunk 7 above)
Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css:115]: I mean, yea, I can probably at like the "unused" color back?

Comment (uncenter) [reply styles/shinigami-eyes/catppuccin.user.css:115]: Huh? Just paste the full declarations from the template over what you have here.

Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css:115]: > Huh? Just paste the full declarations from the template over what you have here.

What you mean by "full declarations" may I ask?

Comment (uncenter) [reply styles/shinigami-eyes/catppuccin.user.css:115]: The variable declarations. https://github.com/catppuccin/userstyles/blob/9b1a1cfb30553b9cc5c98e5a4982793267fac101/template/catppuccin.user.css#L50-L76

File: styles/shinigami-eyes/catppuccin.user.css (hunk 5 above)
Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css:23]: https://github.com/catppuccin/userstyles/pull/1026

File: styles/shinigami-eyes/catppuccin.user.css (hunk 7 above)
Comment (jh-devv) [reply styles/shinigami-eyes/catppuccin.user.css]: https://github.com/catppuccin/userstyles/pull/1026/commits/a09d19f075f5318e964477aa3de8f552fcebb1c5

File: styles/shinigami-eyes/catppuccin.user.css [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+/*
+These are the domains for the sites that Shinigami Eyes works with.
+The origin of the list of these domains is:
+https://github.com/shinigami-eyes/shinigami-eyes/blob/b66e9ad41fa4daef31886b139d8af6f852efcaaa/extension/manifest.json#L15
```
Comment (uncenter) [inline styles/shinigami-eyes/catppuccin.user.css:21]:
```suggestion
The list can be found at https://github.com/shinigami-eyes/shinigami-eyes/blob/b66e9ad41fa4daef31886b139d8af6f852efcaaa/extension/manifest.json#L15-L96.
```

**Review by uncenter — APPROVED** [review]

**Review by isabelroses — APPROVED** [review]
LGTM

**Pull request merged**